.github/workflows/manual_update_development.yml

```diff
@@ -22,3 +22,25 @@ jobs:
           git fetch
           git reset --hard origin/development
           mdbook build -d docs
+
+  job_two:
+    needs: job_one
+    name: Wait for Website Update
+    runs-on: ubuntu-latest
+    steps:
+    - name: Sleep and Run
+      id: wait-deploy
+      run: |
+        echo "Sleeping for 30"
+        sleep 30
+        
+  job_three:
+    needs: [job_one, job_two]
+    name: Check for Broken Links
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check for Broken Links
+        id: link-report
+        uses: docker://ghcr.io/threefoldfoundation/website-link-checker:latest
+        with:
+          args: 'https://www2.manual.grid.tf -w 404'
```

Comment:
Why only check for 404? I think we're going to miss a lot of potential issues like this.

Reply:
We can set it to "-w all" then. But there will be many false positives though, since muffet is very sensitive. It might not be efficient to scan the websites with this. 

I set it to 404 only since that is what really affects the users. For e.g. 403 and 429 are not really affecting links. While checking the TF Manual, only 404 errors were relevant in checking the links.

Let me know what you guys think @A-Harby @scottyeager

Reply:
I set the link checker with those parameters: "-e 404 -w all".

Thus we will get alerts when links with 404 codes are there, and we can also consult all other codes as warnings.

Note that once [this PR](https://github.com/threefoldtech/info_grid/pull/300) is merged, the manual shouldn't have any 404 errors.

Reply:
#300 has been merged and there is no more 404 errors from deadlinkchecker.com.

I think we could merge this PR.